includes/templates/admin/edit-story.php

```diff
@@ -76,8 +78,8 @@
 	),
 	'/web-stories/v1/users/?' . build_query(
```

Comment:
@swissspidy Do we really need to preload the author dropdown? This seems like a lot to load on every editor load.